packages/fx-core/src/component/deploy/azureFunctionDeployDriver.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureResourceInfo, DeployStepArgs, DriverContext } from "../interface/buildAndDeployArgs";
+import { TokenCredentialsBase } from "@azure/ms-rest-nodeauth";
+import { AzureDeployDriver } from "./azureDeployDriver";
+import { DeployExternalApiCallError } from "../error/deployError";
+import { AxiosResponseWithStatusResult } from "../../common/azure-hosting/interfaces";
+import { Service } from "typedi";
+import { StepDriver } from "../interface/stepDriver";
+
+@Service("deploy/azureFunction")
```

Comment:
The official name is azureFunctions, please add "s" for that name

Reply:
Ok, but those driver names are still in draft. They will be modified after the project setting yaml file is finalized